### PR TITLE
Extracted uuid to bundle

### LINKED ISSUE #12532: [Task] Move UUID to UuidBundle within bundles folder in pimcore/pimcore repository
### Improvement description

Move `UUID` to `UuidBundle` within bundles folder in `pimcore/pimcore` repository

### PR DESCRIPTION
**Pull request by aashan10**
## Changes in this pull request  
Resolves #12532

### CODE REVIEW

**Review by mattamon — CHANGES_REQUESTED** [review]
Can you please check this changes according to the refactoring tips too?

**Comment by mattamon** [comment]
@aashan10 can you please merge in the latest 11.x updates, so the tests run trough again?

**Review by mattamon — APPROVED** [review]

File: bundles/UuidBundle/config/pimcore/default.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pimcore_uuid:
+
+    # UUID instance identifier. Has to be unique throughout multiple Pimcore instances. UUID generation will be automatically enabled if a Instance identifier is provided (do not change the instance identifier afterwards - this will cause invalid UUIDs)
+    instance_identifier:  abcde
```
Comment (dvesh3) [inline bundles/UuidBundle/config/pimcore/default.yaml]: Is it needed?

File: bundles/UuidBundle/src/PimcoreUuidBundle.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+
+class PimcoreUuidBundle extends AbstractPimcoreBundle
+{
+    use PackageVersionTrait;
+
+    public function getComposerPackageName(): string
+    {
+        return 'pimcore/uuid-bundle';
+    }
```
Comment (dvesh3) [inline bundles/UuidBundle/src/PimcoreUuidBundle.php]:
```suggestion

```
this is not required as it remains in the core

**Review by dvesh3 — CHANGES_REQUESTED** [review]
- [x]  Add core migration to mark the bundle installed.
- [x]  Dao class mapping should be moved too https://github.com/pimcore/pimcore/blob/83142b53cfd5a791e3c6167470a66336fd03da22/config/dao-classmap.php#L157-L158

- [x] Update docs that feature requires bundle to be installed https://github.com/pimcore/pimcore/blob/6c986c25fd2dc2c969840f2f0f05fd00afae31e2/doc/Development_Documentation/19_Development_Tools_and_Details/19_UUID_Support.md#L1

**Review by dvesh3 — APPROVED** [review]

**Pull request merged**